config_templates/gretel/synthetics/natural-language.yml

```diff
@@ -8,7 +8,7 @@ name: "natural-language-gpt"
 models:
   - gpt_x:
       data_source: "__temp__"
-      pretrained_model: "EleutherAI/gpt-neo-125m"
+      pretrained_model: "gretelai/mpt-7b"
```

Comment:
Do changes to yaml blueprints need to go through the dev branch before being merged to the main?

Reply:
Right, notebooks are handled differently, But you have changes to the config template and (now) use case cards in this PR. I guess I don't understand the develop -> main processes for this repo.

Reply:
Changes that go directly to `main` in this repo won't be reflected in development environments (e.g., Console localhost and console-eng). For changes that could break things in Console (such as reworking use cases) we like to put through develop first so we can review.